### DISCUSSION TITLE
[FIXED]: `roadrunner-cli` can't unpack the `roadrunner-2.10.0-linux-amd64.deb`

### DISCUSSION THREAD

**Discussion by amahbadi** [roadrunner-server/roadrunner#1135] [General]
The following error occurs after downloading the roadrunner by Laravel Octane, Although the version is set to 2.9
**Laravel version: 8.37**
_"laravel/octane": "^1.0",
"spiral/roadrunner": "^2.9"_
**In Factory.php line 97:Can not open the archive "roadrunner-2.10.0-linux-amd64.deb"**
Does anyone have an idea what the problem is?

**Comment by rustatian** [comment]
@amahbadi Hey 👋🏻. This issue was fixed in the latest [`roadrunner-cli` library release](https://github.com/spiral/roadrunner-cli/releases/tag/v2.2.0)

**Comment by amahbadi** [comment]
Hi @rustatian ,Thank you for your prompt response! :ok_hand:
